debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -96,6 +67,21 @@ public void onEntryChosenForEviction(PreparedStatement statement) {
                 }
             });
 
+    public Instant getCurrentTimestamp() throws SQLException {
```

Comment:
For connectors like Oracle, if we're going to resolve this, it will need to be provided the offsets as connectors like Oracle will need to call a function based on the flashback SCN since the time across all table selects is identical for this connector.

Reply:
checking it...